lms/security.py

```diff
@@ -122,6 +126,11 @@ def get_policy(path: str):
                 partial(get_lti_user_from_bearer_token, location="form")
             )
 
+        if path in {"/email/settings", "/email/unsubscribe"}:
```

Comment:
When I look at this I'm both ashamed and proud of the change I made here to explicitly link endpoints to policies. 

Maybe as this grows we could look into making it less error prone (based on route instead of path?, some clever config-like structure?)

Anyway, outside of the scope of this.